src/index.js

```diff
@@ -1,13 +1,51 @@
+// eslint-disable-next-line
 import _ from 'lodash';
-import './styles/style.css'
+import './styles/style.css';
 
-function component() {
-  const element = document.createElement('div');
+const list = document.getElementById('list');
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+const toDos = [
+  {
+    description: 'Wash car',
+    completed: false,
+    index: 1,
+  },
 
-  return element;
-}
+  {
+    description: 'Play fifa',
+    completed: false,
+    index: 2,
+  },
 
-document.body.appendChild(component());
\ No newline at end of file
+  {
+    description: 'Cook',
+    completed: false,
+    index: 3,
+  },
+];
+
+list.innerHTML = `
+  <article>
+    <h2>Today's todo </h2>
+    <iconify-icon icon="bx:refresh"></iconify-icon>
+  </article>
+  <hr>
+  <form>
+    <input type="text" id="new-todo" class="no-outline" placeholder="Add to your list">
+    <button type="submit" id="add-todo" value><iconify-icon icon="uil:enter"></iconify-icon></button>
+  </form>
+  <hr>
+`;
+
```

Comment:
- [ ] Kindly sort the array of tasks first before you display it as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.).